Analyzer/plugins/Analyzer.cc

```diff
@@ -2337,17 +2337,19 @@ void Analyzer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   desc.addUntracked("CalcSystematics",true)->setComment("Boolean to decide  whether we want to calculate the systematics");
   desc.addUntracked("GlobalMaxEta",1.0)->setComment("Cut on inner tracker track eta");
   desc.addUntracked("GlobalMinPt",55.0)->setComment("Cut on pT    at PRE-SELECTION");
+  desc.addUntracked("GlobalMinNOH",8)->setComment("Cut on number of (valid) track pixel+strip hits");
```

Comment:
this is 10 now, no?

Reply:
10 is the number of measurement (NOM) which is different than the number of hits (NOH)

Reply:
Ahh ok NOH is not use anymore, just delete this line (the high purity had a stronger cut on this than what we used)